### PR TITLE
fix(web.domain): change status display on my domain

### PR DESCRIPTION
**Pull request by stif59100**
ref:MANAGER-9564

Signed-off-by: stif59100 <steeve.vanderstocken@ovhcloud.com>

| Question         | Answer
| ---------------- | ---
| Branch?          | `release/components-w6` 
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | MANAGER-9564
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description

I change column for the array of domains name

### CODE REVIEW

File: packages/manager/apps/web/client/app/domain/list/template.html [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+    >
+        <span
+            class="oui-badge"
+            data-ng-class="{
+                'oui-badge_warning': $value === $ctrl.DOMAIN_TECH_STATE.UNLOCKED,
+                'oui-badge_success': $value === $ctrl.DOMAIN_TECH_STATE.LOCKED,
+                'oui-badge_sold-out': $value === $ctrl.DOMAIN_TECH_STATE.UNAVAILABLE,
+
+
+
```
Comment (anooparveti) [inline packages/manager/apps/web/client/app/domain/list/template.html:119]:
```suggestion
```

File: packages/manager/apps/web/client/app/domain/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -1,4 +1,36 @@
...
+  "domains_status_pending_delete": "Supprimé",
+  "domains_status_outgoing_transfer": "Transfert sortant",
+  "domains_status_dispute": "Procédure en cours",
+  "domains_tech_state_not_suspended": "Actif",
+  "domains_tech_state_suspended": "Inactif",
+  "domains_tech_state_locked": "Protégé",
+  "domains_tech_state_unlocked": "Déprotégé",
+  "domains_tech_state_unavailable": "Non applicable",
+  "domains_dns_type_hosted": "Chez Ovhcloud",
+  "domains_dns_type_external": "Hors Ovhcloud",
```
Comment (anooparveti) [inline packages/manager/apps/web/client/app/domain/translations/Messages_fr_FR.json:28]:
```suggestion
  "domains_dns_type_hosted": "Chez OVHcloud",
  "domains_dns_type_external": "Hors OVHcloud",
```

File: packages/manager/apps/web/client/app/domain/list/template.html [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+        data-type="string"
+        data-type-options="$ctrl.statusColumnOptions"
+        data-sortable
+        data-searchable
+        data-filterable
+    >
+        <span
+            class="oui-badge"
+            data-ng-class="{
+                'oui-badge_error': $value === $ctrl.DOMAIN_STATUS.DELETED || $ctrl.DOMAIN_STATUS.PENDING_DELETE ,
```
Comment (y4nnL) [inline packages/manager/apps/web/client/app/domain/list/template.html:52]:
```suggestion
                'oui-badge_error': $value === $ctrl.DOMAIN_STATUS.DELETED || $value === $ctrl.DOMAIN_STATUS.PENDING_DELETE ,
```

File: packages/manager/apps/web/client/app/domain/list/template.html [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+                'oui-badge_success': $value === $ctrl.DOMAIN_STATUS.OK,
+                'oui-badge_warning': $value === $ctrl.DOMAIN_STATUS.RESTORABLE || $ctrl.DOMAIN_STATUS.EXPIRED || $ctrl.DOMAIN_STATUS.OUTGOING_TRANSFER,
```
Comment (y4nnL) [inline packages/manager/apps/web/client/app/domain/list/template.html:54]:
```suggestion
                'oui-badge_warning': $value === $ctrl.DOMAIN_STATUS.RESTORABLE || $value === $ctrl.DOMAIN_STATUS.EXPIRED || $value === $ctrl.DOMAIN_STATUS.OUTGOING_TRANSFER,
```

File: packages/manager/apps/web/client/app/domain/list/template.html [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+        data-searchable
+        data-filterable
+    >
+        <span
+            class="oui-badge"
+            data-ng-class="{
+                'oui-badge_warning': $value === $ctrl.DOMAIN_TECH_STATE.SUSPENDED,
+                'oui-badge_success': $value === $ctrl.DOMAIN_TECH_STATE.NOT_SUSPENDED,
+
+
```
Comment (anooparveti) [inline packages/manager/apps/web/client/app/domain/list/template.html:83]: **nitpick:** Remove unnecessary lines.
```suggestion
```

**Review by anooparveti — APPROVED** [review]

**Review by y4nnL — APPROVED** [review]

File: packages/manager/apps/web/client/app/domain/list/template.html [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+            class="oui-popover-button"
+            oui-popover="{{::'domains_suspension_state_tooltip_' + $value | translate}}"
+            oui-popover-placement="right"
+        ></button>
+        <button
+            data-ng-if="$value === $ctrl.DOMAIN_SUSPENSION_STATE.NOT_SUSPENDED"
+            type="button"
+            class="oui-popover-button"
+            oui-popover="{{::'domains_suspension_state_tooltip_' + $value | translate}}"
+            oui-popover-placement="right"
```
Comment (mohammed-zahaf) [inline packages/manager/apps/web/client/app/domain/list/template.html:91]:
```suggestion
            data-oui-popover="{{::'domains_suspension_state_tooltip_' + $value | translate}}"
            data-oui-popover-placement="right"
```

File: packages/manager/apps/web/client/app/domain/list/template.html [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+        <span
+            class="{{ 'oui-badge ' + $ctrl.DOMAINS_BADGES_TRANSFERT_LOCK_STATE[$value] }}"
+            data-translate="{{ 'domains_transfert_lock_status_' + $value }}"
+        ></span>
+        <button
+            data-ng-if="$value === $ctrl.DOMAIN_TRANSFERT_LOCK_STATE.LOCKED"
+            type="button"
+            class="oui-popover-button"
+            oui-popover="{{::'domains_transfert_lock_status_tooltip_' + $value | translate}}"
+            oui-popover-placement="right"
```
Comment (mohammed-zahaf) [inline packages/manager/apps/web/client/app/domain/list/template.html:112]:
```suggestion
            data-oui-popover="{{::'domains_transfert_lock_status_tooltip_' + $value | translate}}"
            data-oui-popover-placement="right"
```

File: packages/manager/apps/web/client/app/domain/list/template.html [hunk 8]
```diff
@@ -0,0 +1,150 @@
...
+        ></button>
+        <button
+            data-ng-if="$value === $ctrl.DOMAIN_TRANSFERT_LOCK_STATE.UNLOCKED"
+            type="button"
+            class="oui-popover-button"
+            oui-popover="{{::'domains_transfert_lock_status_tooltip_' + $value | translate}}"
+            oui-popover-placement="right"
```
Comment (mohammed-zahaf) [inline packages/manager/apps/web/client/app/domain/list/template.html:119]:
```suggestion
            data-oui-popover="{{::'domains_transfert_lock_status_tooltip_' + $value | translate}}"
            data-oui-popover-placement="right"
```

File: packages/manager/apps/web/client/app/domain/list/template.html [hunk 9]
```diff
@@ -0,0 +1,150 @@
...
+            class="{{ 'oui-badge ' + $ctrl.DOMAINS_BADGES_STATUS[$value] }}"
+            data-translate="{{ 'domains_status_' + $value }}"
+        >
+        </span>
+        <button
+            data-ng-if="$value === $ctrl.DOMAIN_STATUS.RESTORABLE || $value === $ctrl.DOMAIN_STATUS.DISPUTE || $value === $ctrl.DOMAIN_STATUS.EXPIRED"
+            type="button"
+            class="oui-popover-button"
+            oui-popover="{{::'domains_status_tooltip_' + $value | translate}}"
+            oui-popover-placement="right"
```
Comment (mohammed-zahaf) [inline packages/manager/apps/web/client/app/domain/list/template.html:64]:
```suggestion
            data-oui-popover="{{::'domains_status_tooltip_' + $value | translate}}"
            data-oui-popover-placement="right"
```

File: packages/manager/apps/web/client/app/domain/list/template.html [hunk 10]
```diff
@@ -0,0 +1,150 @@
...
+        <a
+            data-ng-href="{{:: $ctrl.getServiceNameLink($row) }}"
+            data-ng-bind="::$value"
+        ></a>
+        <button
+            data-ng-if="$ctrl.isIdn($value)"
+            type="button"
+            class="oui-popover-button"
+            oui-popover="{{::$ctrl.convertDomainToPunycode($value)}}"
+            oui-popover-placement="right"
```
Comment (mohammed-zahaf) [inline packages/manager/apps/web/client/app/domain/list/template.html:43]:
```suggestion
            data-oui-popover="{{::$ctrl.convertDomainToPunycode($value)}}"
            data-oui-popover-placement="right"
```

File: packages/manager/apps/web/client/app/domain/list/template.html [hunk 11]
```diff
@@ -0,0 +1,150 @@
...
+    data-on-page-change="$ctrl.onPageChange($pagination)"
+    data-on-criteria-change="$ctrl.onCriteriaChange($criteria)"
+    data-on-sort-change="$ctrl.onSortChange($sort)"
+    data-columns="$ctrl.columnsConfig"
+    data-on-columns-parameters-change="$ctrl.onColumnChange(id, columns)"
+    data-page="{{ $ctrl.paginationNumber }}"
+    customizable
+>
+    <oui-datagrid-topbar data-ng-if="$ctrl.topbarOptions">
+        <topbar-cta data-options="$ctrl.topbarOptions"> </topbar-cta>
```
Comment (mohammed-zahaf) [inline packages/manager/apps/web/client/app/domain/list/template.html:24]:
```suggestion
        <topbar-cta data-options="$ctrl.topbarOptions"></topbar-cta>
```

**Review by sachinrameshn — APPROVED** [review]

**Review by mohammed-zahaf — APPROVED** [review]

**Pull request merged**